etc/adminhtml/system.xml

```diff
@@ -69,7 +69,7 @@
                     <comment>Enables customers of a bank to easily conduct financial transactions through a bank-operated website (only available in Thailand).</comment>
                     <field id="active" translate="label comment" type="select" sortOrder="140" showInDefault="1" showInWebsite="1" showInStore="1">
                         <label>Enable/Disable</label>
-                        <comment>Enable Omise Internet Banking Solution.</comment>
+                        <comment>Enable Internet Banking payment method.</comment>
```

Comment:
If we want to change `Solution` to `payment method` then we might want to change at the `<group><label></lable></group>` node as well? (it still says `<label>Enable Internet Banking Solution</label>`)